conf/messages

```diff
@@ -157,6 +157,9 @@ confirmEmail.verified.p1=We’ll use this address to contact you about your Capi
 confirmEmail.verified.p2=Your email has not been updated in other government services.
 confirmEmail.resent=Confirmation email sent:
 
+confirmDraftReturn.title=Your Capital Gains Tax on UK property return will be saved for 30 days
+confirmDraftReturn.message=After {0} all the details in your Capital Gains Tax on UK property return will be deleted. After that date, you'll need to start again. We will keep a return that is 'In progress' for 30 days from the day it was last saved. We will only save uploads for 7 days from the last day they were saved.
```

Comment:
we use curly quotes in our messages like the ones here: https://smartquotesforsmartpeople.com/ You can copy and paste the left and right quotes from that page

Reply:
also, shouldn't the sentences all be different `<p>`'s?